<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CenterWidget</class>
 <widget class="QWidget" name="CenterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#ff0;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="firstLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="secondLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="thridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
